### ISSUE TITLE
Improve article 1811.01740

### ISSUE THREAD

**Issue by monniaux** [dginev/ar5iv#38]
**Exact location of issue**
https://ar5iv.org/html/1811.01740#p1.2

**Problem details**
Biblatex is not handled and thus the \bibliography command, which with biblatex just says which bibliography files to use, is interpreted as in ordinary latex, and dumps the contents of the bibliography on the spot, containing commands that do not exist without biblatex and thus are dumped verbatim.

**(Optional) Expected behavior**
If using BibLaTeX, the \bibliography command just says which file to use, and the \printbibliography command prints the bibliography.

**Desktop (please complete the following information)**
Ubuntu Chromium but that's irrelevant

**Comment by dginev** [comment]
Indeed, we need to properly add biblatex support in latexml. cross-referencing [#373](https://github.com/brucemiller/LaTeXML/issues/373)

**Comment by dginev** [comment]
@monniaux I have shipped a basic support for biblatex.sty today and regenerated all articles. Your article now has a recovered HTML bibliography.

Could you describe the remaining problems you see in this+other parts of the page? We'll get it fully supported a step at a time. Thanks!

**Comment by monniaux** [comment]
Figure 4 is broken.

**Comment by dginev** [comment]
We need to add support for listofitems.sty here.

```
Conversion complete: 59 warnings; 39 errors; 11 undefined macros[\bitd, \bite, \bitg, \ways, \bita, \StrChar, \readlist, \bitf, \bitc, \bitb, \setsepchar]; 1 missing file[listofitems.sty]
```
